releasenotes/notes/move-es-version-vars-17c22cfd558a68d5.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
While I can see why we need this for now I think we should work with the folks creating the elk bits to get these vars removed. These should just use whatever the repo provides which in our case will be contained within our own mirror.